index.html

```diff
@@ -0,0 +1,70 @@
+                <p>
+                    Nelson Rolihlahla Mandela (/mænˈdɛlə/;[1] Xhosa: [xolíɬaɬa mandɛ̂ːla]; 18 July 1918 – 5 December 2013) was a South African anti-apartheid revolutionary and political leader who served as the first president of South Africa from 1994 to 1999. He was the country's first black head of state and the first elected in a fully representative democratic election. His government focused on dismantling the legacy of apartheid by tackling institutionalised racism and fostering racial reconciliation. Ideologically an African nationalist and socialist, he served as the president of the African National Congress (ANC) party from 1991 to 1997.
+                </p>
+                <p>
+                    Mandela was born into the Thembu royal family in Mvezo, Union of South Africa. He studied law at the University of Fort Hare and the University of Witwatersrand before working as a lawyer in Johannesburg. There he became involved in anti-colonial and African nationalist politics, joining the ANC in 1943 and co-founding its Youth League in 1944. After the National Party's white-only government established apartheid, a system of racial segregation that privileged whites, Mandela and the ANC committed themselves to its overthrow. He was appointed president of the ANC's Transvaal branch, rising to prominence for his involvement in the 1952 Defiance Campaign and the 1955 Congress of the People. He was repeatedly arrested for seditious activities and was unsuccessfully prosecuted in the 1956 Treason Trial. Influenced by Marxism, he secretly joined the banned South African Communist Party (SACP). Although initially committed to non-violent protest, in association with the SACP he co-founded the militant uMkhonto we Sizwe in 1961 and led a sabotage campaign against the government. He was arrested and imprisoned in 1962, and, following the Rivonia Trial, was sentenced to life imprisonment for conspiring to overthrow the state.
+                </p>
+                <p>
+                    When <b>Nelson Mandela</b> became the first democratically
+                    elected president in South African history in 1994,
+                    the world looked like a very different place. His
+                    election was a symbol of a new birth of freedom
+                    around the world. The fall of the Berlin Wall in
+                    1991 and the end of the Cold War had helped spur a
+                    democratic revolution not only in South Africa but
+                    around the world. Between the start of the 1990s and 2005,
+                    the number of democracies on the planet increased from about
+                    a third of nations to nearly half. Mandela himself was a
+                    global icon not only of democracy but pluralism, and his
+                    triumph seemed to spell the end of an era of authoritarianism
+                    and ethnic nationalism.
+                </p>
```

Comment:
I think it would be better to use semantic tags instead of just divs. So will suggest that you use the article tag on line 24 instead of the div tag to wrap all the information about the legend.